ui/src/app/machines/components/HeaderStrip/ActionFormWrapper/OverrideTestForm/OverrideTestForm.js

```diff
@@ -14,6 +14,56 @@ import FormikField from "app/base/components/FormikField";
 import FormikForm from "app/base/components/FormikForm";
 import MachinesProcessing from "../MachinesProcessing";
 
+const generateFailedTestsMessage = (numFailedTests, selectedMachines) => {
+  if (numFailedTests > 0) {
+    if (selectedMachines.length === 1) {
+      const hostname = selectedMachines[0].hostname;
+      const id = selectedMachines[0].system_id;
+      return (
+        <>
+          <i className="p-icon--warning is-inline"></i>
+          <span>
+            Machine <strong>{hostname}</strong> has
+            <a href={`${process.env.REACT_APP_BASENAME}/#/machine/${id}`}>
+              {` failed ${numFailedTests} ${pluralize(
+                "test",
+                numFailedTests
+              )}.`}
+            </a>
+          </span>
+        </>
+      );
+    } else {
+      return (
```

Comment:
I reckon you could dedupe this and the message above it a bit further...

Reply:
Yeeeaaahh got lazy with that one. Fixed now!